docs/web/using_with_react/translating.mdx

```diff
@@ -1,132 +1,63 @@
 ---
 id: translating
-title: Translating with React
-sidebar_label: Translating with React
+title: Translating
```

Comment:
I am not sure if this is good idea according to SEO

Reply:
Ok, I'll remove it only from sidebar_label - it is redundant there